SA3/qa_web/templates/qa_web/question_display_page.html

```diff
@@ -1,20 +1,59 @@
-{% extends "qa_web/base_navigator.html" %}
+{% extends "qa_web/base.html" %}
 {% load staticfiles %}
 
-{% block content %}
+{% block scripts %}
+    <script src="{% static 'js/Markdown.Converter.min.js' %}"></script>
+    <script src="{% static 'js/Markdown.Editor.min.js' %}"></script>
+    <script src="{% static 'js/Markdown.Sanitizer.min.js'%}"></script>
+    <script src="{% static 'js/base.js'%}"></script>
+{% endblock %}
+
+{% block styles %}
+    <link href="{% static 'css/qa.css' %}" rel="stylesheet"/>
+    <link rel="stylesheet" href="{% static 'css/one-page-wonder.min.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/pagedown.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/base.css' %}" />
+
+{% endblock %}
+
+{% block header %}
+<div class="masthead-content">
+      <div class="container">
+        <h1 class="masthead-heading mb-0">Got Questions</h1>
+        <h1 class="masthead-heading mb-0">? !</h1>
+          <BR><BR><BR>
+      </div>
+</div>
 
+{% endblock %}
+
+{% block content %}
+<BR>
 <div class="col-md-8">
 
-    <ul class="nav nav-tabs nav-justified">
-        <li role="presentation"
-            {% if active_tab == 'latest' %}class="active"{% endif %}>
+    <ul class="nav nav-tabs">
+
+        <li class="active" {% if active_tab == 'latest' %}class="active"{% endif %}>
+            <a href="#latest" data-toggle="tab">
+                <span class="fas fa-clock"></span>
+                    Latest</a>
+        </li>
+<li {% if active_tab == 'hottest' %}class="active"{% endif %}>
```

Comment:
I've implemented a tab for questions that are not answered yet. I could add these tabs' view logic in next sprint.